src/main/kotlin/org/rust/ide/inspections/duplicates/RustDuplicateItem.kt

```diff
@@ -0,0 +1,20 @@
+package org.rust.ide.inspections.duplicates
+
+import com.intellij.psi.PsiElement
+import org.rust.lang.core.psi.RustElementVisitor
+import org.rust.lang.core.psi.RustNamedElement
+
+class RustDuplicateItem {
```

Comment:
Looks like you use this class simply as a wrapper for a function. In this case, you can use an object instead:

```Kotlin
object RustDuplicateItem {
    inline fun <reified T : Any> createInspection( ....
}
```

Though using objects purely for namespaces is somewhat discouraged: https://github.com/yole/kotlin-style-guide/issues/6

Alternatively, you can make this class an abstract inheritor of `RustLocalInspectionTool` and then make `createInspection` function protected. That way you'd be able to enforce privacy better.

Either solution is fine imo.